Source/Core/VideoCommon/ImageWrite.cpp

```diff
@@ -51,23 +51,22 @@ bool TextureToPng(u8* data, int row_stride, const std::string& filename, int wid
 	png_ptr = png_create_write_struct(PNG_LIBPNG_VER_STRING, nullptr, nullptr, nullptr);
 	if (png_ptr == nullptr)
 	{
-		PanicAlert("Screenshot failed: Could not allocate write struct\n");
+		PanicAlertT("Screenshot failed: Could not allocate write struct");
```

Comment:
imo this and the next two alerts don't need to be translated. It's a library issue